pvcompare/era5.py

```diff
@@ -74,39 +86,43 @@ def get_era5_data_from_datespan_and_position(
     chunks=None,
     cds_client=None,
 ):
-    """
-    Send request for era5 data to the Climate Data Store (CDS)
+    r"""
+    Send request for ERA5 data to the Climate Data Store (CDS).
 
     Parameters
     ----------
-    variable: (str or list of str) ERA5 variables to download. If you
-        want to download all variables necessary to use the pvlib, set
+    variable: str or list of str
+        ERA5 variables to download. If you
```

Comment:
```suggestion
        Sets ERA5 variables for download.
```
you couldn't know but it should read:
Sets ERA5 variables for download. If you want to download all variables necessary to use the 'pvlib', set `variable` to 'pvlib'. 
If you want to download variable necessary for using all functionalities of `pvcompare`, set `variable` to 'pvcompare'.
Default: 'pvcompare'.